### PR TITLE
clarified which ISO need to be mounted

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8051

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md [hunk 1]
```diff
@@ -44,11 +44,12 @@ Windows Hello for Business uses asymmetric keys as user credentials (rather than
 
 Sign-in to the domain controller hosting the schema master operational role using enterprise administrator equivalent credentials.
 
-1. Open an elevated command prompt.
-2. Type ```cd /d x:\support\adprep``` where *x* is the drive letter of the DVD or mounted ISO.
-3. To update the schema, type ```adprep /forestprep```.
-4. Read the Adprep Warning.  Type the letter **C** and press **Enter** to update the schema.
-5. Close the Command Prompt and sign-out.
+1. Mount ISO file (or enter the DVD) of Windows Server 2016 or later installation media.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md:47]:
```suggestion
1. Mount the ISO file (or insert the DVD) containing Windows Server 2016 or later installation media.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by MFAISALEMIR** [comment]
Yamaha Kali capek.

**Pull request merged**